spec/factories/teachers.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: teachers
+#
+#  id                     :integer          not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  jti                    :string           not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
```

Comment:
out of curiousity, what is remember_created_at different than created_at?

Reply:
not sure tbh haha; it came with the way Devise was set up